opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -243,6 +243,8 @@ namespace Opm {
                 residual_norms_history_.clear();
                 current_relaxation_ = 1.0;
                 dx_old_ = 0.0;
+                convertInput( iteration, reservoir_state, ebosSimulator_ );
+                ebosSimulator_.model().invalidateIntensiveQuantitiesCache(/*timeIdx=*/0);
             }
```

Comment:
this is not related to this PR, but the invalidation of the intensive quantites cache should probably be moved into `convertInput()`. I'll have a look if nobody objects...